.github/workflows/dotcom-components.yml

```
@@ -0,0 +1,64 @@
+name: Build dotcom-components
+
+on:
+  pull_request:
+    branches:
+      - "**"
+  push:
+    branches:
+      - main
+
+jobs:
+  dotcom-components:
+    if: >-
+      (github.actor != 'dependabot[bot]') &&
+        (github.event.pull_request.head.repo.owner.login == 'guardian' ||
+          github.event_name == 'push')
+    # Required by actions-assume-aws-role
+    permissions:
+      id-token: write
+      contents: read
+
+    name: dotcom-components
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout repo
+        uses: actions/checkout@v2
+
+      - name: Use Node.js
+        uses: guardian/actions-setup-node@main
```

I think we can use the GH action for this now that it supports node version files (this gu action is deprecated).

```suggestion
        uses: actions/setup-node@v3
        with:
            node-version-file: '.nvmrc'
```